src/current/v25.3/user-defined-functions.md

```diff
@@ -113,6 +113,12 @@ SELECT add(3,5) as sum;
 (1 row)
 ~~~
 
+### Call a UDF from a view
+
+You can include UDF calls in a view definition. The function is invoked each time the view is queried. A [materialized view]({% link {{ page.version.version }}/views.md %}#materialized-views) will invoke the function only when the view is created or refreshed.
```

Comment:
Suggest first mention of "view" in this section links to our general docs on views: https://www.cockroachlabs.com/docs/v25.2/views